packages/react/src/components/slide-toggle/slide-toggle.stories.tsx

```diff
@@ -1,15 +1,26 @@
-import { storiesOfSlideToggle } from "dso-toolkit";
-import { storiesOf } from "@storybook/react";
+import type { Meta } from "@storybook/react";
```

Comment:
Deze moet uit `@storybook/web-components` worden geimporteerd.

Reply:
Aha, hier gebeurt het niet. Clear. Chrystal clear. Is there any other kind.

Reply:
😢 ik heb de verkeerde opmerking geplakt. De React Storybook moet uit `@storybook/react` importeren 😭

Reply:
Deze is weer terug uit "@storybook/react"

Reply:
Toppie